src/opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp

```diff
@@ -3116,6 +3126,70 @@ std::shared_ptr<const Python> Schedule::python() const
     return this->python_handle;
 }
 
+
+void Schedule::updateNetwork(std::shared_ptr<Network::ExtNetwork> network, std::size_t report_step) {
+    this->m_network.update(report_step, std::move(network));
+}
+
+const Network::ExtNetwork& Schedule::network(std::size_t report_step) const {
+    return *this->m_network[report_step];
+}
+
+
+void Schedule::handleNODEPROP(const DeckKeyword& keyword, std::size_t report_step) {
+    using NP = ParserKeywords::NODEPROP;
+    auto ext_network = std::make_shared<Network::ExtNetwork>( this->network(report_step) );
+    for (const auto& record : keyword) {
+        const auto& name = record.getItem<NP::NAME>().get<std::string>(0);
+        const auto& pressure_item = record.getItem<NP::PRESSURE>();
+        bool as_choke = DeckItem::to_bool( record.getItem<NP::AS_CHOKE>().get<std::string>(0));
+        bool add_gas_lift_gas = DeckItem::to_bool( record.getItem<NP::ADD_GAS_LIFT_GAS>().get<std::string>(0));
+        Network::Node node{ name };
+
+        if (pressure_item.hasValue(0) && (pressure_item.get<double>(0) > 0))
+            node.terminal_pressure( pressure_item.getSIDouble(0) );
+
+        if (as_choke) {
+            std::string target_group = name;
+            const auto& target_item = record.getItem<NP::CHOKE_GROUP>();
+            if (target_item.hasValue(0))
+                target_group = target_item.get<std::string>(0);
+
+            if (target_group != name) {
+                if (this->hasGroup(name, report_step)) {
+                    const auto& group = this->getGroup(name, report_step);
+                    if (group.numWells() > 0)
+                        throw std::invalid_argument("A manifold group must respond to it's own target");
```

Comment:
Nitpick:
```
...respond to ITS own...
```
Sorry.  didn't see this before.

Reply:
fixed

Reply:
> fixed

Thanks!